src/sentry/api/endpoints/organization_event_details.py

```diff
@@ -35,7 +35,7 @@ def get(self, request: Request, organization, project_slug, event_id) -> Respons
         if event is None:
             return Response({"detail": "Event not found"}, status=404)
 
-        if event.group:
+        if hasattr(event, "for_group") and event.group:
```

Comment:
Just added this in, since transactions can return a `GroupEvent` here. We'll eventually move perf issues to the issue platform, and then we won't need to worry about that case happening anymore.

Reply:
Idk if it's worth leaving a comment, but this seems like it'd be hard to find later when we clean up after moving perf issues.